pkg/controllers/provisioning/scheduling/scheduler.go

```diff
@@ -265,10 +273,24 @@ func (s *Scheduler) Solve(ctx context.Context, pods []*corev1.Pod) Results {
 	}
 }
 
+func (s *Scheduler) updateCachedPodData(p *corev1.Pod) {
+	podData := PodData{
```

Comment:
Want to make the requirements before the struct and then assign to both struct members when making `podData`?

Reply:
Do you think that makes it easier to parse? I'm not really seeing a ton of value though -- though, I'm not 100% what you mean